NEWS.md

```diff
@@ -11,30 +11,240 @@ v1.18.0, Fri Apr 7, 2023
 
 ## Core
 
+- rocr: fix offset calculation
+- rocr: use ofi spinlock functions
+- rocr: minor fixes
+- neuron: convert warn to info for nrt_get_dmabuf_fd not found
+- neuron: check existance of neuron devices during initialization
+- neuron: Add support for neuron dma-buf
+- ze: update ZE to support new driver index specification
+- List variables read from config file
+- Add switch to prefer system-config over environment
+- Add basic system-config support for setting library variables
+- Move peer provider defines into new header
+- rocr: Support asynchronous memory copies
+- rocr: Add support for ROCR IPC
+- rocr: rename rocr data-structures
+- synpaseai: return 0 for host_register and host_deregister
+- fabric: Improve log level of provider mismatch
+- cuda: Allow CUDA IPC when P2P disabled
+- ze: add ZE command list pool to reuse command lists
+- cuda: implement cuda_get_xfer_setting for non cuda build
+- cuda: adjust FI_HMEM_CUDA_ENABLE_XFER behavior
+- cuda.c: Add const to param to remove warning
+- Add IFF_RUNNING check to indicate iface is up and running
+- io_uring support enhancements
+
 ## EFA
 
+- Implement CUDA support on instance that does support GPUDirect RDMA
```

Comment:
Small mistake on our part here, but an important distinction. Sorry 🙂

```suggestion
- Implement CUDA support on instance types that do not support GPUDirect RDMA
```

Reply:
I'll update the NEWS and redo the release package.  I won't update the PR, to let the CI finish.

Reply:
@darrylabbate good catch, thx.